### ISSUE TITLE
Amend media API response schema

### ISSUE THREAD

**Issue by MrTibbles** [meta-network/go-meta#47]
To aid client side parsing of the response payload from the Media API it would be good to alter the way in which the META node handles nullable object responses, especially on the `account` query due to its potential size.

```
type Account {
    performers: [MusicPerformer]!
    labels:     [RecordLabel]!
    composers:  [MusicComposer]!
}
type MusicPerformer {
  identifiers: [PartyIdentifier]!
  name: StringValue
  recordings: [MusicRecordingLink]!
}
```

If the META ID being submitted does not return any performers/labels/composers it returns a single object with no values defined:
```
"performers": [{
  "identifiers": [],
  "name": {
    "value": ""
  },
  "recordings": []
}],
```

When it would be preferred to simply return an empty array.
 ```
performers: []
```

**Comment by MrTibbles** [comment]
Interestingly, with recently merged additions `composers` returns an empty array when none are found.
```
{
  "data": {
    "account": {
      "performers": [
        {
          "identifiers": [],
          "name": {
            "value": ""
          },
          "recordings": []
        }
      ],
      "composers": []
    }
  }
}
```

**Comment by lmars** [comment]
Fixed in #50.
